src/components/Team/chat.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from "react";
+import Modal from "../common/modal/modal";
+import Input from "../common/form-fields/input";
+import { Switch } from "antd";
+import InputArea from "../common/form-fields/input-area";
+import DatePicker from "../common/form-fields/date-picker";
+import Select from "../common/form-fields/select";
+import { useNavigate } from "react-router-dom";
+import { IoCloseOutline } from "react-icons/io5";
+import { GrEmoji } from "react-icons/gr";
+import { PiAtThin } from "react-icons/pi";
+import { CiPaperplane } from "react-icons/ci";
+
+const Chat = ({ isOpen, onSubmit, onClose }: any) => {
+  const navigate = useNavigate();
+  const navigateTo = () => {
+    navigate("/discussion");
+  };
+  return (
+    <div className="mt-6 overflow-auto">
+      {/* upper */}
+      <div className="border-b-2 w-full mb-6">
+        <div className="pb-6 flex items-center justify-between mx-3">
+          <h1 className="text-[#333333] flex-1 justify-center opacity-70 text-center font-semibold text-[18px]">
+            Title
+          </h1>
+          <div>
+            <IoCloseOutline onClick={navigateTo} size={30} />
+          </div>
+        </div>
+      </div>
+      {/* main */}
+      <div className="h-[60vh] ">
+        <div className="h-full overflow-auto px-3">
+          <OtherMessage
```

Comment:
make a constant,
like const messages = [
{
username: "abc",
avatar: "xyz",
type: "other",
content: "he is good"
}, ...]